roles/php/tasks/php-imagick.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: create destination for source
+  command: mkdir -p {{ php_imagick_source_location }}
+      creates={{ php_imagick_source_location }}
+
+- name: copy php imagick tarball
+  copy: src={{ php_imagick_tarball }}
+    dest={{ php_imagick_source_location }}
+
+- name: unpack php imagick source
+  command: tar xf {{ php_imagick_tarball }}
+    chdir={{ php_imagick_source_location }}
+    creates={{ php_imagick_source_path }}
+
+- name: compile php imagick
+  command: '{{ item }} chdir={{ php_imagick_source_path }}
```

Comment:
This needs to `shell:` not `command:` so that we can use the `&&` for the `make && make install` item.